src/CandidateList.js

```diff
@@ -0,0 +1,105 @@
+import React, {useState, useEffect} from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+function CandidateList() {
+    const [candidates, setCandidates] = useState(null);
+    async function getCandidates() {
+        var data = await axios.get(
+            'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates'
+        );
+        setCandidates(data.data);
+    }
+    function Vote(candidate) {
+        async function UpdateVoteCount() {
+            await axios.get(
+                `http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/vote?id=${candidate.id}` 
+            )
+            .then (function(response){
+                alert(candidate.name + '님에게 투표 완료!');
+            })
+            .catch (function(error) {
+                alert('투표 실패');
+            });
+        }
+        UpdateVoteCount();
+    }
+    useEffect(() => {
+        getCandidates();
+    }, [candidates]);
+    if(!candidates){
+        return null;
+    }
+    candidates.sort(function(c1,c2){
+        return c2.voteCount - c1.voteCount;
+    });
+    return(
+        <Wrapper>
+            <Title>CEOS 프론트엔드 13기 개발팀장 투표 창입니다.</Title>
+            <ListWrapper>
+                {candidates.map((candidate, index) => {
```

Comment:
이렇게 해주는게 좀 더 가독성이 좋을 것 같아요 ! [메소드 체이닝 패턴 참고](https://webclub.tistory.com/528)
```suggestion
    return(
        <Wrapper>
            <Title>CEOS 프론트엔드 13기 개발팀장 투표 창입니다.</Title>
            <ListWrapper>
                {candidates
                .sort((c1, c2) => c2.voteCount - c1.voteCount)
                .map((candidate, index) => {
```

Reply:
훨씬 좋네요! 감사합니다 ㅎㅎㅎ